api/generator/generator.go

```diff
@@ -197,6 +198,12 @@ type genSignRequest struct {
 	Request *csr.CertificateRequest `json:"request"`
 	Profile string                  `json:"profile"`
 	Label   string                  `json:"label"`
+	Format  Format                  `json:"format"`
+}
+
+type Format struct {
```

Comment:
You "Format" is an exported type with direct identifier access, which makes it public, you will have to add a line of comment above it or change to lower case "f"